src/v2/System/Router/RouterLink.tsx

```diff
@@ -56,11 +56,28 @@ type RouterLinkMixinProps = BoxProps & {
   textDecoration?: ResponsiveValue<string>
 }
 
+const VALID_FOUND_LINK_PROPS = [
+  "activeClassName",
+  "activeStyle",
+  "exact",
+  "target",
+  "to",
+]
+
+const foundValidatorFn = (prop: string) => {
+  return VALID_FOUND_LINK_PROPS.includes(prop)
+}
+
 export const RouterAwareLink: React.FC<LinkPropsSimple & RouterLinkMixinProps> =
   // TODO: Update styled-components types
   // @ts-ignore
   styled(Link).withConfig({
-    shouldForwardProp: (prop, defaultValidatorFn) => defaultValidatorFn(prop),
+    shouldForwardProp: (
+      prop: string,
+      defaultValidatorFn: (prop: string) => boolean
+    ) => {
+      return defaultValidatorFn(prop) || foundValidatorFn(prop)
```

Comment:
`FOUND` and `found` feel too close to encoding implementation details. can we just say `routerLinkProps`, et?

Reply:
Ah, yeah sure